lib/cert/runner.rb

```diff
@@ -25,6 +31,12 @@ def run
       end
     end
 
+    def expired_certs
+      certificates.select do |certificate|
+        certificate.expires < Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.